sortedList/list.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+typedef struct Node {
+    int data;
+    struct Node* next;
+} Node;
+
+typedef struct SortedList {
+    Node* head;
+} SortedList;
```

Comment:
```suggestion
typedef struct SortedList SortedList;
```
В этой задаче речь шла уже про абстрактные типы данных, так что нельзя было определять структуры в заголовочном файле.